src/Reflection/SignatureMap/Php8SignatureMapProvider.php

```diff
@@ -185,7 +186,10 @@ public function getMethodSignatures(string $className, string $methodName, ?Refl
 	public function getFunctionSignatures(string $functionName, ?string $className, ReflectionFunctionAbstract|null $reflectionFunction): array
 	{
 		$lowerName = strtolower($functionName);
-		if (!array_key_exists($lowerName, $this->map->functions)) {
+		if (
+			!array_key_exists($lowerName, $this->map->functions)
+			|| ($reflectionFunction !== null && $reflectionFunction->getDocComment() !== false && str_contains($reflectionFunction->getDocComment(), '@phpstan-skip-php8-stubs'))
```

Comment:
I didn't want to complicate it with the php8-stubs. The issue is that the stubs look like this:

```php
/** @param array|callable $rest */
function array_intersect_uassoc(array $array, ...$rest): array
{
}
```

which then overrides the newly introduced stub. So I needed to skip them somehow. I suppose an alternative would be to skip php8-stubs where the last parameter is `...$rest`, but I prefer this more explicit way.